tests/unit/db.js

```diff
@@ -39,6 +47,47 @@ function checkValueNotInDb(db, k, done) {
     });
 }
 
+function checkKeyInDB(db, key, cb) {
+    return db.get(key, (err, value) => {
+        if (value || (err && !err.stack.includes('Error: NotFound:'))) {
```

Comment:
Don't we have an attribute `err.notFound` for this? LevelDB returns one I'm pretty sure, so if `ConditionalLevelDB` forwards the error as is it should be there I believe.